storage/storage.go

```diff
@@ -23,88 +25,93 @@ import (
 )
 
 var (
-	ErrObjectAlreadyExists   error = errors.New("the object already exists")
-	ErrObjectDoesNotExist    error = errors.New("the object does not exist")
-	ErrObjectIsDir           error = errors.New("the object is a directory")
-	ErrCopyObject            error = errors.New("error while copying the object")
-	ErrPutObject             error = errors.New("error while putting the object")
-	ErrGetObject             error = errors.New("error while getting the object")
-	ErrHeadObject            error = errors.New("error while getting the object head")
-	ErrDeleteObject          error = errors.New("error while deleting the object")
-	ErrURLJoinPath           error = errors.New("error during url.JoinPath")
-	ErrListObjects           error = errors.New("error while listing objects")
+	// ErrObjectAlreadyExists is returned when an object already exists.
```

Comment:
The one thing I'm not fond of with this approach is the way we lose the formatting alignment in these situations